src/NavBar.js

```diff
@@ -324,9 +325,11 @@ class NavBar extends React.Component {
 
   renderTitle(childState, index:number) {
     const title = this.props.getTitle ? this.props.getTitle(childState) : childState.title;
+    const numberOfLines = this.props.titleNumberOfLines ? this.props.titleNumberOfLines : -1;
```

Comment:
Nicely spotted, I checked it out and when `numberOfLines` is set to `0` no text will be shown in the header. When set to `-1` it will just ignore the `numberOfLines` property. 

Using `Number.isInteger` is cleaner imo but I don't have any strong preference for either.